Changed Release Management Page (#195)
catenax-eV/catenax-ev.github.io: src/pages/release-management.md

Based on the first experiences in live operation, the release management for the next Catena-X ecosystem release will be partially adapted as follows:

## Basic characteristics

> | Topic                               | 101 Pager (R23.09) / Jupiter (Oct. 2024)            |

Reviewer: Is the 101 Pager still up to date? If yes, maybe we should add a reference?

Author: Deleted the 101 Pager reference

> | Upgrade Day for CSP-A / CSP-B / OSP | Synced Upgrade Day (coordinated by Catena-X association) |

Reviewer: I think we should either write the full name, or add a reference to the OM?

Author: Names have been written out and links to the role have been added.

| Grace Period                        | 1 Year                                              |

> | Catena-X & T-X Artefact Versioning       | Semantic Versioning                                 |

Reviewer: ```suggestion
| Catena-X & Eclipse Tractus-X Artefact Versioning       | Semantic Versioning                                 |
```

| Catena-X Ecosystem Release Cycle | 4 Times / Years     | 2 Times / Year                                                                     |
| Catena-X Release Versioning      | Calendar Versioning | Planet Names (e.g. Jupiter)                                                        |
| Catena-X Deployment Gate         |                     | Representatives from each industry partner group to define ecosystem release scope |

> | Backward Compatibility      | No Commitment       | Commitment by�T-X Project                                                          |

Reviewer: ```suggestion
| Backward Compatibility      | No Commitment       | Commitment by the Eclipse Tractus-X Project                                                          |
```

It seems, there is a hidden character.

## Additional notes

[![Release Timeline](@site/static/img/release-schedule.png)](@site/static/img/release-schedule.png)

These principles allow all stakeholders to take advantage of quality-assured dependency and compatibility management, and better predictability for the operational deployment and certification phase.

A group of representatives from each partner will be formed to ensure a coherent product offering for all stakeholders. This group of representatives will be responsible for the governance of the deployment gate after the Jupiter release.

> Versioning with planet names (e.g. Jupiter) is used to improve communication and comprehensibility between releases in the Catena-X association and Tractus-X project as well as the releases in the operative environment. In addition, we are introducing the following two lifecycle states to indicate the validity of our ecosystem releases, which is relevant for the certification period.

Reviewer: ```suggestion
Versioning with planet names (e.g. Jupiter) is used to improve communication and comprehensibility between releases in the Catena-X association and the Eclipse Tractus-X project as well as the releases in the operative environment. In addition, we are introducing the following two lifecycle states to indicate the validity of our ecosystem releases, which is relevant for the certification period.
```

Author: done

- [current] for the months 1-12
- [maintained] for months 13-24 (maintained also contains deprecated versions)
- [deprecated] after month 24

> The versioning of artifacts such as standards, normative documents, KITs or reference implementations is done via semantic versioning.

Reviewer: ```suggestion
The versioning of artifacts such as normative documents (e.g. standards), KITs or reference implementations is done via semantic versioning.
```
